ui/tests/integration/components/selectable-card-test.js

```diff
@@ -0,0 +1,30 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const TOTAL = 15;
+const CARD_TITLE = 'Tokens';
+
+module('Integration | Component meep', function(hooks) {
```

Comment:
hehe, i think you'll want to remove the `meep` here :)

Reply:
Thank you!  I should control find for that specific word before submitting PRs :)